packages/syft/src/syft/node/node.py

```diff
@@ -516,6 +526,7 @@ def named(
                     create_producer=create_producer,
                     queue_port=queue_port,
                     n_consumers=n_consumers,
+                    consumer_service=consumer_service,
```

Comment:
NIT: should we also add consumer_service to also the init of Node Class.

Reply:
Not required as `consumer_service` is part initialized as part of QueueConfig.